CHANGELOG.md

```diff
@@ -116,6 +116,7 @@ PLATFORM SPECIFIC
 
 ### windows
 	+ activated high precission timming to fix inaccurate fps calculations
+	+ allow passing -j<num_jobs> to compileOF.sh
```

Comment:
No worries, force-push should do the trick if you didn't want another commit, pretty sure that's how [rebasing a PR](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) is typically done too.